dotcom-rendering/src/web/components/ShadyPie.tsx

```diff
@@ -10,17 +13,45 @@ params.set(
 );
 params.set('INTCMP', 'shady_pie_open_2019');
 
-export const ShadyPie = () => {
-	return (
-		<a
-			href={`https://support.theguardian.com/uk/subscribe/digital?${params.toString()}`}
-			tabIndex={-1}
-		>
-			<img
-				src="https://uploads.guim.co.uk/2020/10/02/Digisubs_MPU_c1_my_opt.png"
-				width="300"
-				alt=""
+export const ShadyPie = ({
+	sectionName,
+	editionId,
+}: {
+	sectionName: string;
+	editionId: EditionId;
```

Comment:
You could make these props optional and avoid having to pass in default values. I don't think you'd have to change your if statements further down, since they'll cover the case when either property is undefined (you'll get the original shady pie)

```suggestion
	sectionName?: string;
	editionId?: EditionId;
```

Reply:
Thanks for the feedback Chris! Have implemented these suggestions 🙂

Reply:
Thanks!